README.md

```diff
@@ -255,11 +255,15 @@ docker-compose exec anonaddy anonaddy anonaddy:create-user "username" "webmaster
 ```shell
 docker-compose run --entrypoint '' anonaddy gen-dkim
 ```
+
 ```text
-opendkim-genkey: generating private key
-opendkim-genkey: private key written to example.com.private
-opendkim-genkey: extracting public key
-opendkim-genkey: DNS TXT record written to example.com.txt
+generating private and storing in data/dkim/example.com.private
```

Comment:
the output of the tool is less verbose than `opendkim-genkey` so i've added some. happy to revert or change the message.